tachyon/crypto/commitments/kzg/BUILD.bazel

```diff
@@ -25,6 +38,22 @@ tachyon_cc_library(
     deps = [":kzg"],
 )
 
+tachyon_cc_library(
+    name = "kzg_family_test",
+    testonly = True,
+    hdrs = ["kzg_family_test.h"],
+    deps = [
+        ":kzg",
```

Comment:
If it's a `kzg_family_test`, then why is `kzg_family` not included?

Reply:
Yes because when you look at the header, it doesn't include `kzg_family.h`

Reply:
Then isn't `kzg_family_test` misleading as you'd assume from this name that `kzg_family` is used in the test? Would perhaps `kzg_proof_test` work instead?

Reply:
I see, but what I meant was this are something like base test class for both `SHPlonkTest` and `GWCTest`

```c++
class SHPlonkTest : public KZGFamilyTest<PCS> {};
```

```c++
class GWCTest : public KZGFamilyTest<PCS> {};
```

Reply:
If it still makes you confused, I'd like to suggest `KZGTest` instead of `KZGProofTest` considering the name of the directory.

Reply:
I understand your point, but the impact of this naming isn't significant. So, why don't we rename it if we come up with a better idea later?

Reply:
Ok sounds good~